curricula/views.py

```diff
@@ -1074,6 +1074,16 @@ def stage_element(request, stage, format=None):
     serializer = UnitSerializer(unit)
     return Response(serializer.data)
 
+@api_view(['GET', ])
+def stage_standards(request, stage, format=None):
+    try:
+        unit = Unit.objects.get(login_required=False, status=2, stage_name=stage)
+    except MultipleObjectsReturned:
+        logger.exception("Warning - found multiple units referencing the stage %s" % stage)
```

Comment:
Can this case realistically happen? If so, what would it mean?

In general I think it's better to raise exceptions rather than swallowing if its actually an unexpected case.

Reply:
good question. yes, it happens frequently: https://codedotorg.slack.com/archives/C0SUN2VT5/p1578388154147900  

It seems problematic to me too, but has also been declared by Josh C to not be worth doing anything about. Not sure if this is worth tracking at the moment but I didn't comment on this since it seems better to follow the existing pattern, and if we decide to change it we can change it everywhere.

Reply:
This only happens frequently for Professional Learning Courses as far as I know and thats because there is not actually a code studio course associated with the CB course. So we just make up something fake to go in the field and when a course gets cloned it usually doesn't get updated.

Reply:
To add a bit more background, because we're not quite sure how all the platformization stuff will shake out, we don't really know what the future of Curriculum Builder will be just yet -- which makes me want to push pause on this granularity of cleanup task for the moment. If we do decide to double down on CB, then I think it would start making sense to track stuff like this, which we'll notice because it is spamming a slack room which we monitor (so it may not be totally fair to say we are swallowing this error).

Reply:
> This only happens frequently for Professional Learning Courses as far as I know and thats because there is not actually a code studio course associated with the CB course. So we just make up something fake to go in the field and when a course gets cloned it usually doesn't get updated.

thank you for that background @dmcavoy . It sounds like all the noise from the PLC errors would probably hide a problem if a regular code studio course had this problem, so maybe it is a bit worse than I described. I could go either way on whether this is worth tracking then. If anyone feels inclined, it could go in the PLAT project in Jira.

Reply:
I'm still not sure its worth tracking right now.